lxd/main_daemon.go

```diff
@@ -5,10 +5,9 @@ import (
 	"os"
 	"os/exec"
 	"os/signal"
-	"runtime/pprof"
 	"syscall"
-	"time"
 
+	dbg "github.com/lxc/lxd/lxd/debug"
```

Comment:
Why the renamed import?

Reply:
Because there's a "debug" global variable in the main package (which BTW I would personally get rid of, making it non-global). So the go compiler errors out with:

```
10 error debug redeclared as imported package name
previous declaration at /home/free/go/src/github.com/lxc/lxd/lxd/main.go:12 (go-build)
```